packages/manager/src/components/SelectionCard/SelectionCard.spec.js

```diff
@@ -1,205 +1,246 @@
+      const svgs = $$(`${selectionCard} svg`);
+      expect(svgs.length)
+        .withContext(`Incorrect number of icons`)
+        .toBe(3);
+    } else {
+        const svgs = $(`${selectionCard} svg`).isExisting();
+        expect(svgs)
+          .withContext(`Svg icons should not exist`)
+          .toBe(false);
     }
 
-    it('should display selection cards', () => {
-        executeInAllStories(component, childStories, () => {
-            browser.jsClick(selectionCard);
-
-            const selectionCardElems = $$(selectionCard);
-
-            selectionCardElems.forEach(card => expect(card.isVisible()).toBe(true));
-            expect(selectionCardElems.length).toBeGreaterThanOrEqual(1);
-        });
+    expect(fontIcons.length)
+      .withContext(`Font icon should not be displayed`)
+      .toEqual(0);
+  }
+
+  it('should display selection cards', () => {
+    executeInAllStories(component, childStories, () => {
+      const selectionCardElems = $$(selectionCard);
+
+      selectionCardElems.forEach(card =>
+        expect(card.isDisplayed())
+          .withContext(`Selection Card should be displayed`)
+          .toBe(true));
+        expect(selectionCardElems.length)
+          .withContext(`Missing cards to select`)
+          .toBeGreaterThanOrEqual(1);
     });
-
-    it('should display headings on all selection cards', () => {
-        executeInAllStories(component, childStories, () => {
-            const headingElems = $$(heading);
-            headingElems.forEach(e => expect(e.isVisible()).toBe(true));
-        });
+  });
+
+  it('should display headings on all selection cards', () => {
+    executeInAllStories(component, childStories, () => {
+      const headingElems = $$(heading);
+      headingElems.forEach(e =>
+        expect(e.isDisplayed())
+          .withContext(`Heading should be displayed`)
+          .toBe(true));
     });
+  });
 
-    describe('Interactive Example Suite', () => {
-        beforeAll(() => {
-            navigateToStory(component, childStories[0]);
-        });
+  describe('Interactive Example Suite', () => {
+    beforeAll(() => {
+      navigateToStory(component, childStories[0]);
+    });
 
-        it('should display three selection card with one disabled', () => {
-            browser.click(selectionCard);
+    it('should display three selection card with one disabled', () => {
+      $(selectionCard).click();
 
-            const selectionCardElements = $$(selectionCard);
-            const disabledCards = selectionCardElements.filter(e => e.getAttribute('class').includes('disabled'));
+      const selectionCardElements = $$(selectionCard);
+      const disabledCards = selectionCardElements.filter(e => e.getAttribute('class').includes('disabled'));
 
-            expect(disabledCards.length).toBe(1);
-            expect(selectionCardElements.length).toBe(3);
-        });
+      expect(disabledCards.length)
+        .withContext(`Incorrect number of disabled cards`)
```

Comment:
should this be `number of disabled cards` and the following be `number of enabled cards`?